pages.es/linux/zypper.md

```diff
@@ -0,0 +1,28 @@
+# zypper
+
+> Utilidad para la gestión de paquetes en SUSE y openSUSE.
+> Más información: <https://en.opensuse.org/SDB:Zypper_manual>.
+
+- Sincroniza la lista de paquetes y versiones disponibles:
+
+`zypper refresh`
+
+- Instala un nuevo paquete:
+
+`zypper install {{paquete}}`
+
+- Elimina un paquete:
+
+`zypper remove {{paquete}}`
+
+- Actualiza los paquetes instalados a las versiones más recientes disponibles:
```

Comment:
I think this is the correct translation :thinking: 
```suggestion
- Actualiza los paquetes instalados a la versión más reciente disponible:
```

Reply:
In English can read:
> Upgrade installed packages to newest available versions:

since it mention "versions" in plural,that's why I translated like this... But maybe your suggestion looks better...

Reply:
I have checked the translation in Google Translate and looks similar to my version... @navarroaxel what do you think?
> Actualice los paquetes instalados a las versiones más recientes disponibles

Reply:
Sometimes we need to fix it to a fluent sentence in Spanish

Between no more -> entre no más (we have this joke in Argentina).

Reply:
Commited your suggestion :)

Reply:
Thanks @victorhck :rocket: